test/test_faiss_and_milvus.py

```diff
@@ -157,7 +157,7 @@ def test_faiss_write_docs(document_store, index_buffer_size, batch_size):
         original_doc = [d for d in DOCUMENTS if d["content"] == doc.content][0]
         stored_emb = document_store.faiss_indexes[document_store.index].reconstruct(int(doc.meta["vector_id"]))
         # compare original input vec with stored one (ignore extra dim added by hnsw)
-        assert np.allclose(original_doc["embedding"], stored_emb, rtol=0.01)
+        assert np.allclose(original_doc["embedding"] / np.linalg.norm(original_doc["embedding"]), stored_emb, rtol=0.01)
```

Comment:
Could you add a comment here explaining why `/ np.linalg.norm(original_doc["embedding"])` is necessary here and in the next test please?

Reply:
Okay, I have also added a comment in `test_retriever.py` as I'm doing the same there.